source/agora/consensus/EnrollmentManager.d

```diff
@@ -718,6 +718,7 @@ public class EnrollmentManager
                 continue;
             }
             rand_seed = hashMulti(rand_seed, preimage);
+            assert(rand_seed != Hash.init);
```

Comment:
I think this line should be out of the foreach loop because if all the pre-images of the `keys` are PreImageInfo.init, the `rand_seed` will be Hash.init in the end.

Reply:
Good point. I was wondering whether to put this line here, or above the `return`.

Reply:
If it `continue`, then it will not reach the `hashMulti` right ?

Reply:
Correct.